src/frontend/screens/Exchange/ExchangeSettingsBottomSheet.tsx

```diff
@@ -0,0 +1,210 @@
+import React, {useState} from 'react';
+import {View, StyleSheet, Pressable, ScrollView} from 'react-native';
+import {BottomSheetWrapper} from '../../sharedComponents/BottomSheetWrapper';
+import {HeaderText} from '../../sharedComponents/Text/HeaderText';
+import {BodyText} from '../../sharedComponents/Text/BodyText';
+import OrangeStar from '../../images/OrangeStar.svg';
+import GreyLeaf from '../../images/GreyLeaf.svg';
+import {
+  useSetMediaSyncSetting,
+  useGetMediaSyncSetting,
+} from '../../hooks/server/mediaSync';
+import {MediaSyncSetting} from '../../sharedTypes';
+import {BLACK, DARK_GREY, NEW_DARK_GREY, WHITE} from '../../lib/styles';
+import {PrimaryButton, SecondaryButton} from '../../sharedComponents/Buttons';
+import {defineMessages, useIntl} from 'react-intl';
+import {useNavigationFromRoot} from '../../hooks/useNavigationWithTypes';
+
+const m = defineMessages({
+  save: {
+    id: 'screens.ExchangeSettingsBottomSheet.save',
+    defaultMessage: 'Save',
+  },
+  close: {
+    id: 'screens.ExchangeSettingsBottomSheet.close',
+    defaultMessage: 'Close',
+  },
+  everythingTitle: {
+    id: 'screens.ExchangeSettingsBottomSheet.everythingTitle',
+    defaultMessage: 'Exchange Everything',
+  },
+  everythingDesc: {
+    id: 'screens.ExchangeSettingsBottomSheet.everythingDesc',
+    defaultMessage: 'Full size photos and audio. Uses more storage.',
+  },
+  previewsTitle: {
+    id: 'screens.ExchangeSettingsBottomSheet.previewsTitle',
+    defaultMessage: 'Exchange Previews Only',
+  },
+  previewsDesc: {
+    id: 'screens.ExchangeSettingsBottomSheet.previewsDesc',
+    defaultMessage: 'Reduced smaller size photos. No audio included.',
+  },
+});
+
+export const ExchangeSettingsBottomSheet = () => {
+  const {mutate: setMediaSyncSetting} = useSetMediaSyncSetting();
+  const currentSetting = useGetMediaSyncSetting();
+  const {formatMessage: t} = useIntl();
+  const [selected, setSelected] = useState<MediaSyncSetting>('everything');
```

Comment:
initial value should be instantiated based on the actual value
```suggestion
  const [selected, setSelected] = useState<MediaSyncSetting>(() => {
    return currentSetting 
  });
```

Reply:
Of course. Sorry.